src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -121,15 +123,18 @@ function CreatePipelineRun({ intl }) {
       submitError,
       timeout,
       validationError,
-      validTimeout
+      validTimeout,
+      pipelineRunName,
+      validPipelineRunName
     },
     setState
   ] = useState({
     ...initialState,
     namespace: getNamespace(),
     pipelineRef: getPipelineName(),
     params: initialParamsState(null),
-    resources: initialResourcesState(null)
+    resources: initialResourcesState(null),
+    pipelineRunName: `${getPipelineName()}-run-${Date.now()}`
```

Comment:
we should leave the initial state as an empty string, letting the API handle the default value so it's consistent anywhere it's used in future. This is also consistent with the other optional fields where they are left blank / unselected by default.

Reply:
@AlanGreene Thank you for your feedback. I reflected it at https://github.com/tektoncd/dashboard/pull/2345/commits/896d6821d3c6d7e4c12d3427a72b5a22ec691b2d